docs/guides/launch.md

```diff
@@ -57,287 +57,83 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
-
-```
-Successfully generated Typechain artifacts!
-Error: Error: Bytecode length in 32-byte words must be odd
-    at hashL2Bytecode (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/utils.ts:29:15)
-    at computeL2Create2Address (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/utils.ts:53:26)
-    at /Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:50:63
-    at step (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:33:23)
-    at Object.next (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:14:53)
-    at fulfilled (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:5:58)
-error Command failed with exit code 1.
-info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-error Command failed.
-Exit code: 1
-Command: /Users/emilluta/.nvm/versions/node/v16.19.1/bin/node
-Arguments: /opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js compile-and-deploy-libs
-Directory: /Users/emilluta/code/zksync-2-dev/contracts/zksync
-Output:
-
-info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
-error Command failed with exit code 1.
-info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-Error: Child process exited with code 1
-```
-
-**Description**. This means that your bytecode config file has an empty entry: `"bytecode": "0x"`. This happens because
-your `zksync-2-dev/contracts/zksync/package.json`'s dependency on `"@matterlabs/hardhat-zksync-solc"` is outdated. We
-don't expect this error to happen as we've updated to latest version which fixes the problem.
-
-**Solution**. Update your dependency and reinit:
-
-```
-yarn add -D @matterlabs/hardhat-zksync-solc # in the system-contracts folder
-zk clean --all && zk init
-```
-
-On the run, it moved from:
-
-```
-    "@matterlabs/hardhat-zksync-solc": "^0.3.14-beta.3",
-```
-
-to:
-
-```
-    "@matterlabs/hardhat-zksync-solc": "^0.3.15",
-```
-
-### Error: Cannot read properties of undefined (reading 'compilerPath')
-
-**Problem**. `zk init` fails with an error similar to the following:
-
-```text
-Yarn project directory: /Users/<user>/Projects/zksync-era/contracts/system-contracts
-Error: Cannot read properties of undefined (reading 'compilerPath')
-error Command failed with exit code 1.
-```
-
-**Description**. The compiler downloader
-[could not verify](https://github.com/NomicFoundation/hardhat/blob/0d850d021f3ab33b59b1ea2ae70d1e659e579e40/packages/hardhat-core/src/internal/solidity/compiler/downloader.ts#L336-L383)
-that the Solidity compiler it downloaded actually works.
-
-**Solution**. Delete the cached `*.does.not.work` file to run the check again:
-
-```sh
-# NOTE: Compiler version, commit hash may differ.
-rm $HOME/Library/Caches/hardhat-nodejs/compilers-v2/macosx-amd64/solc-macosx-amd64-v0.8.20+commit.a1b79de6.does.not.work
-```
+TODO
```

Comment:
I have 2 scenarios in mind: 
1. You don't have containers running and get this error: 
```
■  error sending request for url (http://127.0.0.1:8545/): error trying to connect: tcp connect error: Connection refused (os error 61)
```
Suggestion to run containers.

2. Failed to remove volumes

```
■  failed to remove path `/home/popzxc/workspace/current/zksync-era/volumes`: Permission denied (os error 13)
```

It's a known problem for linux envs and they have to remove it manually

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/9f144312bb1ed18fcf794fafaf983208225e2b7d